### ISSUE TITLE
AffineTransform

### ISSUE THREAD

**Issue by fisakhan** [tensorflow/addons#496]
**Describe the feature and the current behavior/state.**

**Relevant information**
- Are you willing to contribute it (yes/no):
- Are you willing to maintain it going forward? (yes/no):
- Is there a relevant academic paper? (if so, where):yes
- Is there already an implementation in another framework? (if so, where):https://en.wikipedia.org/wiki/Affine_transformation
- Was it part of tf.contrib? (if so, where):tf.keras.preprocessing.image.apply_affine_transform

**Which API type would this fall under (layer, metric, optimizer, etc.)**

**Who will benefit with this feature?**

**Any other info.**

**Comment by seanpmorgan** [comment]
Hi @fisakhan! Currently Keras is undergoing a preprocessing redesign:
https://github.com/keras-team/governance/pull/10

As I understand it, all keras preprocessing utils will be made compatible with `tf.data` and so tensorflow ops for all of that functionality will land in TF core.

Please feel free to re-open if this doesn't cover your request.
